### ISSUE TITLE
How do you handle time?

### ISSUE THREAD

**Issue by huwd** [openpolitics/uk-taxes#54]
This seems useful.
Wondering how you're approaching time here?

The [calculations](https://github.com/openpolitics/uk-taxes/blob/master/features/support/calculations.rb) would always have to be current. I was thinking of spinning off a gem to or api wrapper for HMRC data, to just capture the bandings, thresholds and rates... but i think that would need to be an array of tax years with figures held as values.

What would that do to your cucumber tests?

Could we wrap them in a context:

Given `I am trying to assess my tax liabilities for the tax year 2019-2020`?

Could then get it to update those whenever we get new bandings?

**Comment by Floppy** [comment]
Yeah, the experiment really didn't get that far. But I think I'd do it with a `Given`, as you say. `Given is the tax year 2019` or somesuch. Do they always change on a tax year basis, or do some things change mid-year? Booze tax, for instance.
